portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -212,7 +213,18 @@
 
 
     <!-- servlet definitions & mappings -->
-
+    <servlet-mapping>
```

Comment:
@Luke-Sikina Can we consolidate this mapping with the mappings on [line 589](https://github.com/cBioPortal/cbioportal/blob/9f6344eaf3323d828f41e5cfa53e454ca7fb62f6/portal/src/main/webapp/WEB-INF/web.xml#L589)

Reply:
Yes, fixed.